src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -32,6 +32,12 @@
 			<NewGroupConversation v-if="canStartConversations" />
 		</div>
 		<template #list>
+			<Button v-if="!preventFindingUnread && unreadTop > 0"
+				class="unread-mention-button unread-mention-button__top"
+				type="primary"
+				@click="scrollTopUnread">
+				{{ unreadTop }} more mentions
+			</Button>
```

Comment:
In the mobile apps we only show it for the bottom and I think that is okay.

Reply:
Ok, for now,  desktop has 2 buttons, and mobile has 1 button :)